### DISCUSSION TITLE
Drive a 12V LED strip with an additional PSu

### DISCUSSION THREAD

**Discussion by HeresyPurger** [sblantipodi/firefly_luciferin#149] [Q&A]
Hi,
First of all, thank you for your work. 
I'm actually considering building an lightning system for my setup and decided to give a try to the DIY solution considering the premium others are asking.

I'm considering using an Arduino ESP32 with the light luciferin firmware ( to avoid the need of opening a tab and log in each time I want to go on a more "static scene" for reading). 
However, I saw that only 5V LED strip are mentionned. My question is: is this a hardware limitation (to much current to be handled by the chip) or software ?
If it's a hardware limitation, does a simple addition of an external power supply is sufficent to be able to drive a 12V LED strip ?

Thank you.

**Comment by sblantipodi** [comment]
Hi, official PCB is designed for 5V but you can run Luciferin no problem on others PCB or on your custom PCB that supports 12V.

Sorry for the late reply ;)

**Reply by HeresyPurger** [reply]
Hi, no problem with it :)

Figured it out yesterday actually.

Thx for the reply.
